assets/js/script.js

```diff
@@ -0,0 +1,164 @@
+const myAPIKey = "f23ee9deb4e1a7450f3157c44ed020e1";
+const currentWeather = $("#current-weather");
+const searchContainer = $("#search-history");
+const forecastContainer = $("#forecast-container");
+
+const constructSearchHistory = function (array) {
+  array.forEach((element) => {
+    historyButton = `<li><button id="search-btn">${element}</button></li>`;
```

Comment:
Make sure your variables are always declared with const or let - never like this!

Reply:
oops